src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
> "Dangerous" because your are opening the Project class here, so everything that someone add here by mistake overwrites the model definition.

No this doesn't overwrite the model definition? This is a nested class, there is no overriding going on.. 

> Misleading in the sense of that your are defining parts of the Project model class in other file...

This is a nested class. Thats different from defining a part of the `Project` class in another file.